substrate/frame/staking/src/tests.rs

```diff
@@ -5967,240 +6142,252 @@ fn force_apply_min_commission_works() {
 
 #[test]
 fn proportional_slash_stop_slashing_if_remaining_zero() {
```

Comment:
Note that nothing in this tests changes, only wrapping the test in a `TestExternalities` and making sure that the ledger used for the tests is bonded.

The test needs to be in a `TestExternalities` context because this PR implements `OnStakingUpdate::on_slash` which requires storage access.